CMake/Install.cmake

```diff
@@ -12,13 +12,96 @@ set(XCBUILD_INSTALL_DEVELOPER_DIR Developer CACHE STRING "Install path for xcbui
 # XCBUILD_INSTALL_TARGET(foo bar qix) installs the CMake targets 'foo', 'bar',
 # and 'qix' into the xcbuild Developer dir (specified by
 # XCBUILD_INSTALL_DEVELOPER_DIR).
+#
+# If USER is specified, also install the targets into CMAKE_INSTALL_PREFIX.
+# Mutually exclusive with USER.
+#
+# If USER_ALIAS is specified, also install xcrun into CMAKE_INSTALL_PREFIX
+# renamed for each target. All targets must be executables. Mutually exclusive
+# with USER.
+#
+# USER_ALIAS allows users to easily access Developer dir tools without adding
+# Developer/usr/bin to $PATH. For example, XCBUILD_INSTALL_ALIAS(plutil USER)
+# will install xcrun into /usr/local/bin/plutil. Because xcrun inspects its file
+# name at runtime, xcrun will find and invoke ${DEVELOPER_DIR}/usr/bin/plutil.
 function (XCBUILD_INSTALL_TARGET)
-  cmake_parse_arguments("" "" "" "" ${ARGN})
+  cmake_parse_arguments("" "USER;USER_ALIAS" "" "" ${ARGN})
+  if (${_USER} AND ${_USER_ALIAS})
+    message(AUTHOR_WARNING "XCBUILD_INSTALL_TARGET was called with mutually exclusive options: USER and USER_ALIAS")
+    set(_USER_ALIAS False)
+  endif ()
   set(TARGETS ${_UNPARSED_ARGUMENTS})
   # ARCHIVE is deliberately omitted; intentionally do not install static
   # libraries (including Windows import libraries).
   # TODO(strager): Handle FRAMEWORK and BUNDLE targets.
   install(TARGETS ${TARGETS}
           RUNTIME DESTINATION ${XCBUILD_INSTALL_DEVELOPER_DIR}/usr/bin
           LIBRARY DESTINATION ${XCBUILD_INSTALL_DEVELOPER_DIR}/usr/lib)
+  if (${_USER})
+    install(TARGETS ${TARGETS}
+            RUNTIME DESTINATION bin
+            LIBRARY DESTINATION lib)
+  endif ()
+  if (${_USER_ALIAS})
+    foreach (TARGET ${TARGETS})
+      get_target_property(TYPE ${TARGET} TYPE)
+      if (${TYPE} STREQUAL EXECUTABLE)
+        # NOTE(strager): Assumes XCBUILD_INSTALL_TARGET(xcrun USER) was (or will
+        # be) called.
+        XCBUILD_INSTALL_PROGRAM_ALIAS(TARGET ${TARGET} ALIASED_TARGET xcrun DESTINATION bin)
+      else ()
+        message(AUTHOR_WARNING "XCBUILD_INSTALL_DEVELOPER_DIR cannot install non-executable (${TYPE}) target ${TARGET} with USER_ALIAS")
+      endif ()
+    endforeach ()
+  endif ()
+endfunction ()
+
+# XCBUILD_INSTALL_PROGRAM_ALIAS(ALIASED_TARGET a DESTINATION d TARGET t)
```

Comment:
Rather than installing an alias for a target, why not just install an alias for a program name? Then, in something like `Aliases/CMakeLists.txt`, just have a list of all of the aliases to install:

    XCBUILD_INSTALL_ALIAS(xcbuild)
    XCBUILD_INSTALL_ALIAS(actool)
    XCBUILD_INSTALL_ALIAS(clang)

I think that'd be simpler overall and would allow installing aliases for tools that are not part of xcbuild itself.